### PR TITLE
IVORY-321-Choose an exemption type

### PR DESCRIPTION
**Pull request by Jozzey**
Created "What is your ivory item?" page + unit tests

### CODE REVIEW

File: test/routes/what-type-of-item-is-it.route.test.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      it('should store the value in Redis and progress to the next route when the third option has been selected', async () => {
+        await _checkSelectedRadioAction(
+          postOptions,
+          server,
+          'Portrait miniature made before 1918 with a surface area less than 320 square centimetres',
+          nextUrl
+        )
+      })
+
+      it('should store the value in Redis and progress to the next route when the forth option has been selected', async () => {
```
Comment (duncannixon) [inline test/routes/what-type-of-item-is-it.route.test.js:184]: Typo (fourth)

File: test/routes/what-type-of-item-is-it.route.test.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        await _checkSelectedRadioAction(
+          postOptions,
+          server,
+          'Item to be sold or hired out to a qualifying museum',
+          nextUrl
+        )
+      })
+
+      it('should store the value in Redis and progress to the next route when the fith option has been selected', async () => {
```
Comment (duncannixon) [inline test/routes/what-type-of-item-is-it.route.test.js:193]: Typo (fifth)

File: test/routes/what-type-of-item-is-it.route.test.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+          server,
+          'Item made before 1918 that has outstandingly high artistic, cultural or historical value',
+          nextUrl
+        )
+      })
+    })
+
+    describe('Failure', () => {
+      it('should display a validation error message if the user does not select an item', async () => {
+        postOptions.payload.ivoryIsIntegral = ''
```
Comment (duncannixon) [inline test/routes/what-type-of-item-is-it.route.test.js:205]: Payload item name incorrect for this route.

File: test/routes/what-type-of-item-is-it.route.test.js [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+  beforeEach(() => {
+    _createMocks()
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('GET: Owner applicant', () => {
```
Comment (duncannixon) [inline test/routes/what-type-of-item-is-it.route.test.js:48]: No need for 'Owner applicant' here

**Review by duncannixon — CHANGES_REQUESTED** [review]
A couple of minor typos but otherwise good stuff :)

**Comment by Jozzey** [comment]
Changes made as per review

**Review by duncannixon — APPROVED** [review]
Good job

**Pull request merged**